server/protocols/jmap-rfc-8621/src/test/scala/org/apache/james/jmap/routes/JMAPApiRoutesTest.scala

```diff
@@ -44,14 +43,81 @@ class JMAPApiRoutesTest extends AnyFlatSpec with BeforeAndAfter with Matchers {
   private val JMAP_API_ROUTE: JMAPApiRoutes = new JMAPApiRoutes()
   private val ROUTES_HANDLER: ImmutableSet[JMAPRoutesHandler] = ImmutableSet.of(new JMAPRoutesHandler(Version.RFC8621, JMAP_API_ROUTE))
 
-  private val REQUEST_OBJECT: String = """{"using":["urn:ietf:params:jmap:core"],"methodCalls":[["Core/echo",{"arg1":"arg1data","arg2":"arg2data"},"c1"]]}"""
-
+  private val REQUEST_OBJECT: String =
```

Comment:
I think some of those constants are similar (if not same) with the ones declared in `EchoMethodContract` right? Should we extract those into the fixture class? (it's a question)

Reply:
Yes we could reuse at Fixture but then we need dependency.WDYT? @mbaechler  @chibenwa

Reply:
Let's keep the constant duplicated IMO.

We wrote extensive unit tests for echo method because we did not have the integration test set up yet.

Now that we have the integration test setup, I would keep the echo method unit tests as they do no harm here (no mocks for instance).